codeReview_7_27_17_1.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+x = input ("Enter a number: ")
+sum = 0
+i = 1
+
+while i <=x:
```

Comment:
yes, done! 
I can't type Korean because I didn't install(?) Korean keyboard here.
(but I prefer to see reviews in Korean to understand better :) )
anyway, I used for loop for the first time, and just changed 'for' to 'while'. 

I need to figure out what is different between while and for loops.